CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# dbt_marketo v0.13.0
+[PR #43](https://github.com/fivetran/dbt_marketo/pull/43) includes the following updates:
+
+## Breaking Change
+- Updated the default configuration for the `marketo__enable_campaigns` and `marketo__enable_programs` variables from disabled to enabled.
+
+- Each variable will enable the following models by default:
+  - `marketo__enable_campaigns` : `marketo__campaigns`, `marketo__email_stats__by_campaign`, the upstream `stg_marketo__campaigns`, in addition to `marketo__programs`, `marketo__email_stats__by_program`, and the upstream `stg_marketo__program` (these last 3 also require the `marketo__enable_programs` variable).
+  - `marketo__enable_programs` : `marketo__programs` and `marketo__email_stats__by_program`, in combination with the `marketo__enable_campaigns` variable.
+
+- Additionally, the `marketo__email_sends` model will now output the following fields which were previously skipped by default now that `marketo__enable_campaigns` is enabled at onset:
```

Comment:
could you include a link to the `marketo__email_sends` docs page so people can see the definitions of the new fields